readme.md

```diff
@@ -152,17 +152,17 @@ class Hello extends Component {
 export default Hello
 ```
 
-Ok let's recap what's going on.
+Let's recap what's going on.
 
 #### What's that HTML doing in my Javascript?
 
 Often times we write out React components in **JSX**.
-* JSX is [an alternate Javascript syntax](http://blog.yld.io/2015/06/10/getting-started-with-react-and-node-js/#.V8eDk5MrJPN) that allows us to write code that strongly resembles HTML. It is eventually transpiled to lightweight JavaScript objects.
+* JSX is [a language that compiles to Javascipt](http://blog.yld.io/2015/06/10/getting-started-with-react-and-node-js/#.V8eDk5MrJPN) that allows us to write code that strongly resembles HTML. It is eventually compiled to lightweight JavaScript objects.
```

Comment:
:+1: Turns out "transpiling" is also technically correct, but the way you have it now is good.